canfoundation/can-signing-protocol-example-web
# Feat/transfer history for given token

**This MR did:**

- New feature  "Check balance of user"
- New feature  "User's token transfer history for given token"
- New feature "Transfer history for given token"

**How to test:**
Run `yarn start` and go to http://localhost:3002/

Input an can account to check balance of user (I already auto fill your can account when you logged in)
<img width="456" alt="Screen Shot 2020-11-27 at 11 27 28 AM" src="https://user-images.githubusercontent.com/24496205/100412676-e84d8c00-30a7-11eb-924e-4e3527a113cb.png">


View transfer history of an token
- Go to http://localhost:3002/tranfer-history
- Enter `Code` and `Symbol` field
- Click `GET HISTORY` button for example as this picture below

<img width="1110" alt="Screen Shot 2020-11-30 at 2 59 47 PM" src="https://user-images.githubusercontent.com/24496205/100584607-e80af600-331e-11eb-8204-a6a8192efc27.png">


View transfer history of an user with given token, just add their can account in `CAN Account` field and token code, token symbol.

<img width="1110" alt="Screen Shot 2020-11-30 at 3 03 02 PM" src="https://user-images.githubusercontent.com/24496205/100584654-f48f4e80-331e-11eb-9a6b-eeec79403db8.png">


I will write readme later when this MR be merged

@duyphanz help me check this again.
